editor/components/icon-button/index.js

```diff
@@ -10,12 +10,13 @@ import './style.scss';
 import Button from '../button';
 import Dashicon from '../dashicon';
 
-function IconButton( { icon, label, className, ...additionalProps } ) {
+function IconButton( { icon, text, label, className, ...additionalProps } ) {
 	const classes = classnames( 'editor-icon-button', className );
 
 	return (
 		<Button { ...additionalProps } aria-label={ label } className={ classes }>
 			<Dashicon icon={ icon } />
+			{ text ? <span>{ text }</span> : null }
```

Comment:
The exact same thing, just replace text with children in order to use this like this:

`<IconButton>MyText</IconButton>`

I've done something similar in #429 but without the `span`(I don't mind keeping the span)